kubernetes-pipeline/templates/jenkins/jenkins-casc-conf.yaml

```diff
@@ -298,18 +267,10 @@ data:
               {{- if .chart.customChart }}
               helm package {{ .chart.name }} -u
               {{- else }}
-              helm repo update
+              helm repo add wso2 https://helm.wso2.com
```

Comment:
Don't we need to run a repo update first?

Reply:
@ThilinaManamgoda it seems that without updating it downloads the charts in Helm v3.

Reply:
@ThilinaManamgoda I tested this setup with WSO2 IS version `5.8.0` pattern 1.

Reply:
@chirangaalwis Yes, but with a **repo update** it caches the chart information locally which avoids subsequent calls to the chart repo.

Reply:
@ThilinaManamgoda will update.